menu_konsolowe.py

```diff
@@ -16,12 +21,19 @@ def customer_board():
         elif user_choice == "5":
             print("5. ANULOWANIE REZERWACJI")
         elif user_choice == "6":
-            print("6. DANE KONTAKTOWE BIURA")
+            with open("dane do printu.txt", "r") as file:
+                lines = file.readlines()
+                fragment = lines[70:86]
```

Comment:
takie 70 i 86 w programowaniu nazywa sie `magic numbers`. Wyobraz sobie ze za tydizen ktso wejdzie do tego pliku tekstowego i doda 2 linie i wszystkie te lines[0:10] itd straca sens i trzeba je bedzie recznie naprawiac i sprawdzac. To wszystko powinno byc w osobnych plikach a taki przynajmniej mam pierwszy pomysł :) https://en.wikipedia.org/wiki/Magic_number_(programming)